kliff/uq/mcmc_utils.py

```diff
@@ -68,18 +61,13 @@ def mser(
 def autocorr(chain: np.ndarray, *args, **kwargs):
     """Use ``emcee`` package to estimate the autocorrelation length.
 
-    Parameters
-    ----------
-    chain: np.ndarray (nwalkers, nsteps, ndim,)
-        Chains from the MCMC simulation. The shape of the chains needs to be
-        (nsteps, nwalkers, ndim). Note that the burn-in time needs to be discarded prior
-        to this calculation
-    args, kwargs
-        Additional positional and keyword arguments of ``emcee.autocorr.integrated_time``.
-
-    Returns
-    -------
-    float or array:
+    Args:
+        chain: Chains from the MCMC simulation. The shape of the chains needs to be
```

Comment:
For `np.ndarray` type, it still is good to add what they mean. Like it is here, `(nwalkers, nsteps, ndim,)` is very informative. The automated tool we use can only tell it is an np.ndarray. 

So, can you please add these back -- here and for other np.ndarray cases?